ui/lib/core/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "core",
+  "keywords": [
+    "ember-addon"
+  ],
+  "dependencies": {
+    "ember-auto-import": "*",
+    "ember-cli-htmlbars": "*",
+    "ember-cli-htmlbars-inline-precompile": "*",
+    "ember-cli-babel": "*",
+    "ember-radio-button": "*",
+    "ember-composable-helpers": "*",
+    "ember-cli-string-helpers": "*",
+    "ember-truth-helpers": "*",
+    "ember-concurrency": "*",
+    "base64-js": "*",
+    "Duration.js": "*"
```

Comment:
This is one of the nice things about in-repo-addons IMO - package version management is done in the host application's `package.json`, you add deps here to help the build step, but there's no need to specify versions separately.